code/notificationbuilder/src/main/java/com/adobe/marketing/mobile/notificationbuilder/internal/PushTemplateConstants.kt

```diff
@@ -134,6 +135,15 @@ internal object PushTemplateConstants {
         const val RATING_UNSELECTED_ICON = "adb_rate_unselected_icon"
         const val RATING_SELECTED_ICON = "adb_rate_selected_icon"
         const val RATING_ACTIONS = "adb_rate_act"
+        internal object TimerKeys {
```

Comment:
nit: can we either alter the other keys to be inside their own objects or have this with the suffix `TIMER_ALTERNATE_TITLE` etc for uniformity?

Reply:
I am thinking the other way around :) 
We should keep Template specific keys in its own object.
We can take this later in our cleanup steps.